website/docs/docs/dbt-cloud/dbt-cloud-api/service-tokens.md

```diff
@@ -1,38 +1,62 @@
 ---
-title: "Service Account Tokens"
+title: "Service account tokens"
 id: "service-tokens"
+description: "Service account tokens help you define permissions for securing access to your dbt Cloud account and its projects."
 ---
 
-## Overview
-Service Account Admin API tokens are like [User API tokens](user-tokens), but they belong to an
-_account_, rather than _a user_.
+## About service tokens
 
-As such, they may be suitable for system-level integrations that do not run on behalf of any one user. Service Account Admin tokens may only be created or modified by users with Account Admin (Enterprise plan) or Owner (Team plan) permissions on an account. Service Account tokens may be revoked if they become lost or compromised.
+Service account tokens enable you to securely authenticate with the dbt Cloud API by assigning each token a narrow set of permissions that more precisely manages access to the API. While similar to [User API tokens](user-tokens), service account tokens belong to an account rather than a user.
 
-### Service Account Admin API tokens
+You can use service account tokens for system-level integrations that do not run on behalf of any one user. Assign any role available in dbt Cloud to your service account token. For more information about roles, see "[Enterprise Permissions](docs/dbt-cloud/access-control/enterprise-permissions)".
 
-<Changelog>
+## Generating service account tokens
 
- - Service Account Admin tokens were launched in February, 2021
+In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new token. Create and save your token somewhere safe.
 
-</Changelog>
+:::caution Note
 
-Service Account Admin API tokens currently have full read+write access to an account, so please use them with caution.
+You will not be able to view this token again after generating it, so store the token somewhere safe for later use.
 
-#### Generating a Service Account Admin Token
+:::
+## Permissions for service account tokens
 
-In the Account Settings view of dbt Cloud, you can click on the Service Account tokens page and generate a new Account Admin token.  Create and save your token. Please note that you will not be able to view this token again after its is generated, so store it somewhere safe for later use.
+You can assign service account tokens any role available in dbt Cloud. When you assign a role to a token, you will also be able to choose whether to grant that permissions to all projects in the account or to specific projects.
 
-### Metadata Only Service Account Tokens
+The following permission list summarizes each role. For more details about the permissions in these roles, see "[Enterprise permissions](/docs/dbt-cloud/access-control/enterprise-permissions)."
```

Comment:
I wonder if we should decouple the list of permission sets from the list of "enterprise permissions". Non-Enterprise accounts can now use these permission sets when creating Service Tokens, so documenting them under "Enterprise Permissions" is kind of confusing!

Reply:
@drewbanin Who all can use these permission sets? Is it Teams and Enterprise accounts?

Reply:
@saimaddali and @drewbanin I think I will address the Enterprise permissions page separately from this PR. I'd like to do it thoughtfully and not squeeze it in with this release 🥂 

Sai, maybe we can talk more about Drew's great point when we sync up this week?

Reply:
(I did make separate lists for service tokens so hopefully that will help)

Reply:
I will create a follow up issue for this suggestion!